app/controllers/idv/how_to_verify_controller.rb

```diff
@@ -56,12 +56,14 @@ def self.enabled?
     end
 
     def self.step_info
+      #puts "*** yo I got called"
```

Comment:
Extremely scientific debugging.

TIL from this that we call `step_info` probably a dozen times on page loads, though it's not expensive.